ee/modules/110-istio/hooks/dataplane_metadata_exporter.go

```diff
@@ -196,11 +198,18 @@ func dataplaneMetadataExporter(input *go_hook.HookInput) error {
 		}
 	}
 
+	ignored := 0
+	withoutSidecar := 0
+	total := 0
```

Comment:
1. There should be a prefix like `flantTelemetry`.
2. The var name must be informative like `totalPodsInCluster`.

The total one, btw, doesn't have sense.

Reply:
done